src/webapp/README.md

```diff
@@ -0,0 +1,118 @@
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This sample demonstrates a Python Flask web application that is both protected by Microsoft identity platform and accesses Microsoft Graph as the user by using the Microsoft Authentication Library (MSAL) for Python..
+
+![A browser screenshot on a page showing a response from Microsoft Graph](./app.png)
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- An Azure Active Directory (Azure AD) tenant and required permissions to create App registrations.
+- Python 3.8+
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                                  | Notes                                                                                              |
+|:----------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                     | `python-webapp`                                                              | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**  | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**            | **Web**                                                                      | Required value for this sample. <br/> Enables the required and optional settings for the app type. |
+| **Redirect URI**             | `http://localhost:5000/auth/redirect`                                        | Required value for this sample.                                                                    |
+| **Client secret**            | _Value shown in Azure portal_                                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **Implicit grant & hybrid flows** | _None selected_                                                         | This sample does not use the implicit grant or hybrid flows.                                       |
```

Comment:
Leave out anything they don't have to change from default.

Reply:
Intent of "This sample" is unclear to me.

Reply:
I used that term ("this sample") from your other rows in the table where you used "this sample" to reference this thing you're looking at.  Does that help?

As you requested, I should leave this line out anyhow as _None selected_ is the default anyway.  So, I'll update this to remove this line.

Reply:
Updated, "default" values have been removed from this table.